win32ss/user/user32/controls/appswitch.c

```diff
@@ -164,22 +164,28 @@ void CompleteSwitch(BOOL doSwitch)
 
 BOOL CALLBACK EnumerateCallback(HWND window, LPARAM lParam)
 {
-   HICON hIcon;
+#define ICON_TIMEOUT 80 // in milliseconds
+#define ICON_RETRY_COUNT 10
+   HICON hIcon = NULL;
+   LRESULT ret;
+   UINT uFlags = SMTO_ABORTIFHUNG | SMTO_NORMAL, cRetry;
```

Comment:
```suggestion
   UINT uFlags = SMTO_ABORTIFHUNG | SMTO_NORMAL;
   UINT cRetry;
```

Reply:
Since you don't seem to change the values of uFlags, you could just put these flags directly in the SendMessageTimeout function call.